Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -922,14 +898,13 @@
     "Context": {
       "ReferenceDateTime": "2017-12-01T13:37:00"
     },
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
         "Text": "van 11:00 tot 3",
         "Type": "timerange",
         "Value": {
-          "Timex": "(T11,T15,PT4H)",
+          "Timex": "(T11:00,T15,PT4H)",
```

Comment:
This seems incorrect/inconsistent. Can you validate the behaviour is the same as in English?
If it is, we may want to create an issue to standardize the behaviour (change across languages).

Reply:
yes, as discussed in previous comment, also in English when the time is written as "11:00" is parsed as T11:00 and when is written as "11" or "11 am" is parsed as "T11".
So, "from 11:00 to 3" is parsed as (T11:00,T15,PT4H)